tftrt/examples/cpp/image-classification/README.md

```diff
@@ -0,0 +1,54 @@
+# TF-TRT example for conversion in C++
+
+## Introduction
+
+This directory contains example code to demonstrate TF-TRT conversion using the C++ API.
+
+### Limitations
+- Freezing the model is not implemented correctly, the current example only works with frozen graphs.
+- Some of the intermediate steps are possibly inefficient. This can influence the conversion time. The inference runtime is not affected.
```

Comment:
Comment about conversion efficiency unnecessary? I'd be more worried about memory than time. e.g. is user going to go into hard system lock if they try to convert a large nlp model?